corehq/apps/linked_domain/tests/test_update_other_fixtures.py

```diff
@@ -0,0 +1,107 @@
+from django.test import TestCase
+from corehq.apps.fixtures.dbaccessors import (
```

Comment:
This module was deleted a while ago. Are the tests in this file running on Github Actions?

Reply:
I've since modified the configuration to have tests run. It appears these tests have long been broken. Updated the code to handle the SQL changes now.